horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/TestExport.java

```diff
@@ -69,12 +74,12 @@ public void updateRefs() {
         }
     }
 
-    public void updateExperimentsVariableId(int oldVarId, int newVarId) {
+    public void updateExperimentsVariableId(String variableName, int newVarId) {
         if (experiments != null && !experiments.isEmpty()) {
             for (ExperimentProfile experiment : experiments) {
                 if (experiment.comparisons != null && !experiment.comparisons.isEmpty()) {
                     experiment.comparisons.forEach(c -> {
-                        if (c.variableId == oldVarId)
+                        if (c.variableName.equals(variableName))
```

Comment:
I had to use the `variableName` to properly link experiment comparisons to variables as there wouldn't be other ways since the variables are created from scratch and the id is known only before this step (the user does not know about that when uploading the schema)